### PR TITLE
Add supported module standard docs

### PR DESCRIPTION
**Pull request by camfindlay**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/tests            export-ignore
+/docs             export-ignore
+/.gitattributes   export-ignore
+/.travis.yml      export-ignore
+/.travis          export-ignore
+/.scrutinizer.yml export-ignore
```
Comment (assertchris) [inline .gitattributes]: Could I trouble you to put a trailing newline here?

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,25 @@
 # See https://github.com/silverstripe-labs/silverstripe-travis-support for setup details
+language: php
 
-language: php 
 php:
-  - 5.4
+    - 5.3
+    - 5.4
```
Comment (assertchris) [inline .travis.yml]: `5.5`, `5.6` plz

File: .travis.yml [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
-    - poppler-utils
-
 env:
   - DB=MYSQL CORE_RELEASE=3.1
-  - DB=MYSQL CORE_RELEASE=3
+
+matrix:
+  include:
+    - php: 5.5
+      env: DB=PGSQL CORE_RELEASE=3
```
Comment (assertchris) [inline .travis.yml]: Probably want to test `PGSQL` with `3`, `3.1`, and `3.2`; and `MYSQL` with `3`, and `3.1` here.

File: .travis.yml [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
 
 sudo: false
 
-addons:
-  apt:
-    packages:
-    - poppler-utils
-
 env:
   - DB=MYSQL CORE_RELEASE=3.1
```
Comment (assertchris) [inline .travis.yml]: Default env should be `3.2`

Comment (assertchris) [inline .travis.yml]: Default env should be `3.2`

File: .travis.yml [hunk 5]
```diff
@@ -23,7 +27,19 @@ before_script:
...
   - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss
   - cd ~/builds/ss
+  - composer install
 
 script:
   - ($HOME/bin/tika-rest-server &) &> /dev/null
   - vendor/bin/phpunit --verbose textextraction/tests/
+  - wget https://scrutinizer-ci.com/ocular.phar
+  - git remote rm origin
+  - git remote add origin git@github.com:silverstripe-labs/silverstripe-textextraction.git
```
Comment (assertchris) [inline .travis.yml]: Should probably remove cover coverage reporting stuff (so this remote and ocular stuff).

**Comment by camfindlay** [comment]
@assertchris updated based on feedback.

File: .travis.yml [hunk 6]
```diff
@@ -1,21 +1,35 @@
...
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.1
+    - php: 5.6
+      env: DB=PGSQL CORE_RELEASE=3.2
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.3
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.2
```
Comment (assertchris) [inline .travis.yml]: Can remove this ↑
It's already covered by the default env + PHP versions...

**Comment by chillu** [comment]
Needs a rebase

**Comment by robbieaverill** [comment]
@camfindlay if you could allow contributors push access I can rebase this and get it merged

**Comment by dhensby** [comment]
> @camfindlay if you could allow contributors push access I can rebase this and get it merged

you can merge by hand and resolve any conflicts / make any changes needed during merge:

```sh
git clone https://github.com/silverstripe/silverstripe-textextraction
cd silverstripe-textextraction
git checkout master
git fetch origin pull/19/head:pull/19
git merge --no-ff --no-commit pull/19
# resolve conflicts, make changes
git commit -m 'Merge pull request #19 from camfindlay/feature/make-supported'
```

**Comment by robbieaverill** [comment]
Yeah true, ok

**Comment by dhensby** [comment]
:)

**Pull request merged**